streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -313,36 +282,29 @@ public void shouldNotFlushAndThrowOnCloseDirty() {
 
         final double expectedCloseTaskMetric = 1.0;
         verifyCloseTaskMetric(expectedCloseTaskMetric, streamsMetrics, metricName);
-
-        EasyMock.verify(stateManager);
     }
 
     @Test
     public void shouldNotThrowFromStateManagerCloseInCloseDirty() {
-        EasyMock.expect(stateManager.changelogOffsets()).andStubReturn(Collections.emptyMap());
-        stateManager.close();
-        EasyMock.expectLastCall().andThrow(new RuntimeException("KABOOM!")).anyTimes();
-        EasyMock.replay(stateManager);
+        doThrow(new RuntimeException("KABOOM!")).when(stateManager).close();
 
         task = createStandbyTask();
         task.initializeIfNeeded();
 
         task.suspend();
         task.closeDirty();
 
-        EasyMock.verify(stateManager);
+        verify(stateManager, atLeastOnce()).close();
     }
 
     @Test
     public void shouldSuspendAndCommitBeforeCloseClean() {
```

Comment:
I think for this test you need to verify the calls on the mock with `InOrder` (see https://site.mockito.org/javadoc/current/org/mockito/InOrder.html).

Reply:
I am unsure what needs to be verified in order? The current version of the test is 

```java
@Test
public void shouldNotThrowFromStateManagerCloseInCloseDirty() {
    doThrow(new RuntimeException("KABOOM!")).when(stateManager).close();

    task = createStandbyTask();
    task.initializeIfNeeded();

    task.suspend();
    task.closeDirty();
}
```

And from what I can tell its testing that the `RuntimeException` ie never thrown when `task.closeDirty()` is called (`task.closeDirty() indirectly calls `stateManager.close()`). The only mock is the `doThrow` so there is nothing to really order?

Reply:
Actually, my comment is on line 305, i.e., method `shouldSuspendAndCommitBeforeCloseClean()` and not `shouldNotThrowFromStateManagerCloseInCloseDirty()`. I think there you should verify that `stateManager.checkpoint` is called before `stateManager.close()`. But since the current test does also not check the order and this PR is about migration, you can consider this an optional request.